.git-blame-ignore-revs

```diff
@@ -1,2 +1,5 @@
 # Adjusted indentation to 2 spaces
 ba3e1355fe8b2612b1f48d26ab68f6902efde691
+
+# Replaced many `ccall` by `@ccall`
+4850d2b8e7b3c6da47836a926f727bfabe4a6654
```

Comment:
So I guess for that to work we'd need to "merge" (not rebase, not squash)

Reply:
I dropped this for now. It seems easier to do another small PR afterwards than to keep this hash right during all of the rebasing I am during currently.